evap/contributor/templates/contributor_evaluation_form.html

```diff
@@ -63,7 +63,15 @@ <h5 class="card-title">{% trans 'Course data' %}</h5>
             <div class="card-body">
                 <div class="d-flex">
                     <h5 class="card-title me-auto">{% trans 'Evaluation data' %}</h5>
-                    {% if not evaluation.allow_editors_to_edit %}
+                    {% if evaluation.allow_editors_to_edit %}
+                        {% if not manager %}
```

Comment:
you can skip this check. it is the contributor form and not the staff form, so managers won't use it :)